read_buffer/src/chunk.rs

```diff
@@ -170,6 +170,20 @@ impl Chunk {
         Self::base_size() + table_data.size()
     }
 
+    /// Return the estimated size, for each column in the specific table
+    /// if no such table exists in this chunk, an empty Vec is returned
+    ///
+    /// The estimated size for each column in this table. Note there
+    /// may be multiple entries for each column.
```

Comment:
This comment looks like it's repeating itself, I think it can be consolidated unless I'm missing something? Also, is the multiple entries bit still true since you got rid of the need for `coalesce`?

```suggestion
    /// Return the estimated size for each column in the specific table.
    /// Note there may be multiple entries for each column.
    ///
    /// If no such table exists in this chunk, an empty Vec is returned.
```